src/components/BannedCharacters/BannedCharacters.js

```diff
@@ -0,0 +1,61 @@
+import { h } from 'preact';
+import classNames from 'classnames';
+
+const FONT_SIZE = 18;
+
+const BannedCharacters = ({ bannedCharacters = [], highlightedCharacters = [] }) => (
+    <div class="banned-characters">
```

Comment:
I'd also check `bannedCharacters.length` and if `0` – won't render component at all

Reply:
Sure